scripts/actions/fetch-and-deserialize.js

```diff
@@ -0,0 +1,85 @@
+const { vendorRequest, vendorGetRequest } = require('./utils/vendor-request');
+const AdmZip = require('adm-zip');
+const deserializedHtml = require('./deserialize-html');
+const { write } = require('to-vfile');
+const vfile = require('vfile');
+const createDirectories = require('../utils/migrate/create-directories');
+
+const localesMap = {
+  'ja-JP': 'jp',
+};
+
+const projectId = process.env.TRANSLATION_VENDOR_PROJECT;
+
+const fetchFileURIs = async (batchUid) => {
+  const {
+    response: {
+      data: { files },
+    },
+  } = await vendorRequest(
+    'GET',
+    `/job-batches-api/v2/projects/${projectId}/batches/${batchUid}`
+  );
+  return files.map((file) => file.fileUri);
+};
+
+const fetchTranslatedFilesZip = async (fileUris, locale) => {
+  const fileUriStr = fileUris.reduce((str, uri) => {
+    return str.concat(`&fileUris[]=${uri}`);
+  }, '');
+
+  const localeIdStr = `localeIds[]=${locale}`;
+
+  return vendorGetRequest(
+    `/files-api/v2/projects/${projectId}/files/zip?${localeIdStr}${fileUriStr}`
+  );
```

Comment:
I think we need the HTTP method as the first parameter.

```suggestion
  return vendorGetRequest(
    'GET'
    `/files-api/v2/projects/${projectId}/files/zip?${localeIdStr}${fileUriStr}`
  );
```

Reply:
So I needed to create a separate `vendorRequest` function (I should probably rename it something more specific) because I didn't want the function to return `response.json()`